R/new_matrixfit.R

```diff
@@ -389,7 +391,25 @@ new_matrixfit <- function(cf,
     stopifnot(cf$nrObs == 1)
     model_object <- NParticleModel$new(cf$Time, parind, sign.vec, ov.sign.vec)
   }
-  
+
+  if(model == 'n_particles') {
+    initial_guess = function(corr, summands, t1, t2){
+      t1p1 <- t1 + 1
+      t2p1 <- t2 + 1
+      par <- numeric(2*summands)
+
+      par[1] <- 1.13
```

Comment:
Wait, it takes various parameters and just ignores them all and puts in fixed numbers? Isn't that what `par.guess` is for? The `initial_guess` function should just try to make a guess if the user has not specified anything.